design/dhw_tank_temperature.md

```diff
@@ -0,0 +1,96 @@
+# Control Verification Algorithm -- DHW_tank_temperature
```

Comment:
I added these markdown documents for your review. I plan to delete these after review.